src/v2/view-builder/views/phone/EnrollAuthenticatorDataPhoneView.js

```diff
@@ -120,9 +99,14 @@ const Body = BaseForm.extend({
     return uiSchemas;
   },
 
+  updateConditionalText() {
+    this.el.querySelector('.okta-form-subtitle').innerText = this.subtitle();
+    this.el.querySelector('input[type="submit"]').value = this.save();
+  },
+
   initialize() {
     BaseForm.prototype.initialize.apply(this, arguments);
-    this.listenTo(this.model, 'change:authenticator.methodType', this.render.bind(this));
+    this.listenTo(this.model, 'change:authenticator.methodType', this.updateConditionalText);
```

Comment:
Avoid a full render in order to retain the show/hide animation of the Extension field.